### PR TITLE
Update Blog “understanding-hpe-greenlake-identity-access-management”

### PR DESCRIPTION
**Pull request by lenihan01**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by choukden — CHANGES_REQUESTED** [review]
Hi John, 

Thank you very much for taking into account the new guidelines for the naming convention of the HPE GreenLake for Private Cloud Enterprise.
 
Just a few comments:

The Role and Responsibility table is not formatted correctly. If you look at the preview post [here](https://deploy-preview-1666--hpe-dev-portal.netlify.app/blog/understanding-hpe-greenlake-identity-access-management/) you will notice formatting issue with the table. I believe there is a space character and a "backslash" character just after the line "Consumption Analytics Contributor" that are causing the issue.

You may also want to change the Responsibility for the **Private Cloud Tenant Owner** into Administer HPE GreenLake for Private Cloud Enterprise: Virtual Machines dashboard ....

 Once you incorporated the comments, I will merge the PR to move the updated post to production on HPE Dev portal.

If you agree, after publication of the updated post, I can add the editor's note at the top of the blog post:
**Editor’s Note – NAME CHANGE: HPE GreenLake for Private Cloud is now part of HPE GreenLake for Private Cloud Enterprise.** 

Let me know your opinion?

Best regards,
Denis

**Comment by lenihan01** [comment]
Hi @choukden ,
Thanks for spotting those two additional changes which were needed. I have pushed a change to hopefully address them and will check once the preview is available.

Please feel free to add the editor's note.

File: content/blog/understanding-hpe-greenlake-identity-access-management.md [hunk 1]
```diff
@@ -56,11 +58,11 @@ Roles are available for the services that are available within the tenant. The f
...
+| Private Cloud Tenant Owner                      | Administer HPE GreenLake for Private Cloud Enterprise: Virtual Machines dashboard<br>Manage scheduling and activity<br>Manage infrastructure<br>Manage provisioning                                                                                    |
 | Private Cloud Tenant Contributor                | Manage self-service VMs and app provisions                                                                                                                                                                                |
 |
-| **HPE GreenLake for Containers**                |                                                                                                                                                                                                                           |
+| **HPE GreenLake for Private Cloud: Containers on Virtual Machines**                |                                                                                                                                                                                                                           |
```
Comment (choukden) [inline content/blog/understanding-hpe-greenlake-identity-access-management.md:65]: Hi John,
Shouldn't it be "HPE GreenLake for Private Cloud Enterprise: Containers on Virtual Machines" ?

File: content/blog/understanding-hpe-greenlake-identity-access-management.md [hunk 2]
```diff
@@ -195,7 +187,7 @@ The space looks like this to the users:
 
 #### IT Department
 
-The IT department space would need access to HPE GreenLake for Private Cloud, HPE GreenLake for ML Ops and HPE GreenLake for Containers resources. This would allow users in this department to manage the resources running in both services. Users in this department require admin roles for the selected resources.
+The IT department space would need access to HPE GreenLake Enterprise Private Cloud: Virtual Machines, HPE GreenLake for ML Ops and HPE GreenLake for Private Cloud Enterprise: Containers resources. This would allow users in this department to manage the resources running in both services. Users in this department require admin roles for the selected resources.
```
Comment (choukden) [inline content/blog/understanding-hpe-greenlake-identity-access-management.md:190]: This should be: HPE GreenLake for Private Cloud Enterprise: Virtual Machines

File: content/blog/understanding-hpe-greenlake-identity-access-management.md [hunk 3]
```diff
@@ -103,47 +102,40 @@ The invited user receives an email inviting them to join the tenant. Once the us
...
+![](/img/iam-document-example-1-tenant-1-1a.jpg "Example 1: ACME Corp.")
 
-Customer ACME Corp. has users spread around the globe. The users' email addresses all take the form of [xyz@acmecorp.com.](mailto:xyz@acmecorp.com.) ACME Corp. has several departments that are also distributed around the globe. ACME Corp. wishes to use HPE GreenLake Private Cloud to create and manage virtual machines and containers on behalf of the various departments. They also wish to use HPE GreenLake for ML Ops to examine their internal data and perform AI operations upon it and use HPE GreenLake for containers to run a sales application.
+Customer ACME Corp. has users spread around the globe. The users' email addresses all take the form of [xyz@acmecorp.com.](mailto:xyz@acmecorp.com.) ACME Corp. has several departments that are also distributed around the globe. ACME Corp. wishes to use HPE GreenLake Private Cloud Enterprise to create and manage virtual machines and containers on behalf of the various departments. They also wish to use HPE GreenLake for ML Ops to examine their internal data and perform AI operations upon it and use HPE GreenLake for containers to run a sales application.
```
Comment (choukden) [inline content/blog/understanding-hpe-greenlake-identity-access-management.md:138]: This should be ...to use HPE GreenLake for Private Cloud Enterprise to create....

(the "for" is missing).

**Review by choukden — CHANGES_REQUESTED** [review]
Hi John,

Just a couple of minor comments.

The table looks good now.

Once you incorporate the small changes I will push the update to the HPE dev portal. And I will then add the Editor's note.

Best regards,
Denis

**Pull request merged**